.github/ISSUE_TEMPLATE/release.md

```diff
@@ -11,8 +11,8 @@ Release vX.X.X
 
 ## Checklist
 
+- [ ] Update metadata & clusterctl-settings
 - [ ] Update docs, (compatibility table, usage etc) .
 - [ ] Create tag.
 - [ ] Update the created draft release to include (BREAKING Changes, Important Notes).
 - [ ] Publish the release.
-- [ ] OPTIONAL: create release branch.
```

Comment:
it's saying is OPTIONAL,
whenever we need it.

Reply:
@avorima Are you okay with not supporting release branches? 
I'm okay if there's general consensus.